cfy_manager/main.py

```diff
@@ -343,12 +353,14 @@ def db_node_add(**kwargs):
                      default=False)
 @argh.decorators.arg('-a', '--address', help=DB_NODE_ADDRESS_HELP_MSG,
                      required=True)
+@argh.decorators.arg('-i', '--node-id', help=DB_NODE_ID_HELP_MSG,
```

Comment:
Isn't this only required if running the command on a manager?

Reply:
Yes.
So it runs also on the db nodes?

Reply:
The remove is, yes.